rollbar/contrib/django/middleware.py

```diff
@@ -18,11 +18,16 @@
 
 import rollbar
 
+import django
 from django.core.exceptions import MiddlewareNotUsed
-from django.core.urlresolvers import resolve
 from django.conf import settings
 from django.http import Http404
 
+if django.VERSION >= (1, 10):
```

Comment:
Thanks for the link. That's definitely something to consider since a method could change to a property and vice-versa. I'm going to file an internal ticket to discuss with the team.